docs/_includes/_nav.njk

```diff
@@ -1,24 +1,24 @@
 <header>
   <div class="logo-bar">
     <a href="{{ '/' | url }}">
-      <img src="{{ '/brand/logo/svg/pfe-icon-white-shaded.svg' | url }}" alt="PatternFly Elements - Home">
+      <img src="{{ '/brand/logo/svg/pf-icon-white-shaded.svg' | url }}" alt="PatternFly Elements - Home">
```

Comment:
`/brand/logo/svg/` icons have not been renamed from `pfe` to `pf`.  Should be renamed as well?